### PR TITLE
Bug 1133338 - Use vertical homescreen on flatfish

### PR DESCRIPTION
**Pull request by shamenchens**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[shamenchens Sherman Chen (shamenchens)](https://github.com/shamenchens) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5f643ce312e8)

**Comment by try-server-hook** [comment]
[shamenchens Sherman Chen (shamenchens)](https://github.com/shamenchens) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e2aeccbeed14)

**Comment by try-server-hook** [comment]
[shamenchens Sherman Chen (shamenchens)](https://github.com/shamenchens) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a58dca524503)

File: build/homescreen-manager.js [hunk 1]
```diff
@@ -217,6 +217,14 @@ function loadHomescreen(config) {
     'build', 'default-homescreens.json');
   var customize = utils.getJSON(defaultConfig);
 
+  // Load device specific configuration
+  var deviceConfig = utils.resolve(
+    utils.joinPath('build', 'config', config.GAIA_DEVICE_TYPE,
+      'homescreens.json'), config.GAIA_DIR);
```
Comment (RickyChien) [inline build/homescreen-manager.js]: Just use ```utils.getFile(config.GAIA_DIR, 'build', 'config', config.GAIA_DEVICE_TYPE, 'homescreens.json')``` instead.

**Comment by try-server-hook** [comment]
[shamenchens Sherman Chen (shamenchens)](https://github.com/shamenchens) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e3d121bb8a6c)

**Pull request merged**